packages/react-charting/UnitTests/SankeyChartUT.test.tsx

```diff
@@ -17,8 +17,7 @@ import {
 import { SLink, SNode } from '../src/types/IDataPoint';
 import { IMargins } from '../src/utilities/index';
 
-const env = require('../config/tests');
-const runTest = env === 'TEST' ? describe : describe;
+const runTest = describe;
```

Comment:
Seems to a mistake. Could you update it to
const runTest = env === 'TEST' ? describe : describe.skip;